docs/guides/account-maps.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Account Maps
+---
+
+# Account Maps
+
+If you're coming from Ethereum, you must be familiar with Solidity's `mapping` data structure which allows you to map key-value pairs of different types. This mapping is stored in the smart contract's data, and is very similar to other `Map`-like data structures.
+
+Solana's Account model, as we know, unlike Ethereum, separates program data and its state into different accounts. Learn more about Solana's Account model [here][AccountCookbook].
+
+This presents us with a problem many Solana developers face, which is implementing a `Map`-like logic into their programs. One way to do this is using [PDAs][CreatePDA], or Program Derived Addresses.
+
+## Deriving Accounts using PDAs
+
+In this method, we can use an account's public key as a seed to derive the account it is mapped to. 
+
+We'll take an example of a **Blog** program to demonstrate how this would be implemented.
```

Comment:
Let's explain we're creating a blog, and blogs have posts! We need a map, and we can use PDAs!

Something more eloquent than that